src/wp-admin/includes/file.php

```diff
@@ -1946,6 +1946,75 @@ function copy_dir( $from, $to, $skip_list = array() ) {
 	return true;
 }
 
+/**
+ * Moves a directory from one location to another.
+ *
+ * Recursively invalidates OPcache on success.
+ *
+ * If the renaming failed, falls back to copy_dir().
+ *
+ * Assumes that WP_Filesystem() has already been called and setup.
+ *
+ * @since 6.2.0
+ *
+ * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
+ *
+ * @param string $from      Source directory.
+ * @param string $to        Destination directory.
+ * @param bool   $overwrite Overwrite destination.
+ *                          Default is false.
```

Comment:
```suggestion
 * @param bool   $overwrite Optional. Whether to overwrite the destination directory if it exists.
 *                          Default false.

```
For consistency with the description of this parameter in filesystem classes' `::move()` method.